Adjust README for Source Code Example (#13)
GDP-ADMIN/gen-ai-examples: examples/gen-ai-hello-world/README.md

> # GenAI RAGO Example Using Binary Version of SDK

Reviewer: ```suggestion
# GenAI RAGO Example Using the Binary Version of the SDK
```

> This is an example of how to use the gllm-pipeline library to build a simple RAG pipeline using binary version of SDK.

Reviewer: ```suggestion
This is an example of how to use the gllm-pipeline library to build a simple RAG pipeline using the binary version of the SDK.
```

> See other examples in [gen-ai-hello-world](../gen-ai-hello-world).

Reviewer: ```suggestion
See other examples in [gen-ai-internal-hello-world](../gen-ai-internal-hello-world).
```
other example harusnya yg internal (?)
atau mungkin bisa diganti pake parent directory "examples" aja (?)

Reviewer: ```suggestion
See the examples using source code version of SDK on [gen-ai-internal-hello-world](../gen-ai-internal-hello-world).
```

atau mungkin dijelasin dikit other example yg dimaksud itu yg source code versionnya

Author: oh betul, harusnya pakai parent directory. Nanti kuganti.

Author: Updated di https://github.com/GDP-ADMIN/gen-ai-examples/pull/13/commits/7ba7350a8ed64a0b12c1892692e2f6ca218fc541 ya

> See other examples in [gen-ai-hello-world](https://github.com/GDP-ADMIN/gen-ai-examples).

Reviewer 2: should be `gen-ai-examples`?
